common/achievement/template.ts

```diff
@@ -168,11 +168,11 @@ export async function checkTemplateConsistencyBeforeActivating(template: Achieve
     logger.info(`Checking AchievementTemplate(${template.id}) for consistency`, { template, group });
 
     // ---- Template Metadata -----
-    if (!template.title || !template.description || !template.subtitle) {
+    if (!template.title || !template.description) {
         throw new PrerequisiteError(`AchievementTemplates need a name, description and subtitle`);
     }
 
-    const actionFields = [!!template.actionName, !!template.actionRedirectLink, !!template.actionType];
+    const actionFields = [!!template.actionName, !!template.actionType];
```

Comment:
There might be actions that don't have a link, like waiting for course approval.